distributed/cli/dask_submit.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+
+import click
+from tornado import gen
+
+from tornado.ioloop import IOLoop
+
+from distributed import rpc
+from distributed.cli.utils import check_python_3
+
+logger = logging.getLogger('distributed.dask_submit')
+
+
+@click.command()
+@click.argument('scheduler_address', type=str, required=True)
+@click.argument('filepath', type=str, required=True)
+def submit(scheduler_address, filepath):
+    @gen.coroutine
+    def f():
+        scheduler_host, remote_client_port = scheduler_address.split(':')
```

Comment:
Is this the address of the scheduler or the address of the `dask-remote` process?